stages/apps/lead/sonarqube.tf

```diff
@@ -1,13 +1,14 @@
 data "vault_generic_secret" "sonarqube" {
-  path = "lead/aws/${data.aws_caller_identity.current.account_id}/sonarqube"
+  count = var.enable_sonarqube ? 1 : 0
+  path  = "lead/aws/${data.aws_caller_identity.current.account_id}/sonarqube"
```

Comment:
fix: don't query vault for this secret when `enable_sonarqube` is set to `false`